src/test/java/teammates/test/cases/browsertests/InstructorFeedbackEditPageUiTest.java

```diff
@@ -214,10 +214,32 @@ private void testEditSessionAction() throws Exception {
         feedbackEditPage.clickEditSessionButton();
         editedSession.setInstructions(new Text("Made some changes"));
         feedbackEditPage.editFeedbackSession(editedSession.getEndTime(), editedSession.getStartTime(),
-                                        editedSession.getInstructions(), editedSession.getGracePeriod());
+                                        editedSession.getInstructions(), editedSession.getGracePeriod(), true);
 
         String expectedString = "The end time for this feedback session cannot be earlier than the start time.";
         feedbackEditPage.verifyStatus(expectedString);
+
+        ______TS("test discard changes to feedback session");
+
+        // Make changes to Feedback session
```

Comment:
Hmm I would think it would not be good since html files are pretty difficult to check manually. Are there possible workarounds such as checking both pages after clicking edit button? Your solution might not be sufficient as well. If you reload the page at the point at which you check against `instructorFeedbackEditDiscardChangesOk.html`, The `Responses visible from` option changes from `Publish manually` to ` Immediately`.